platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -6,64 +6,109 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/** Label placement relative to its geometry. `line` can only be used on LineStrings and Polygons. */
```

Comment:
These documentation comments need the same transformations we put in for properties in #6076.